projects/GPGame/README.md

```diff
@@ -0,0 +1,17 @@
+# GP_Engine
+
+Link to Production Pipeline document : https://docs.google.com/document/d/16_K5LYEYgaHoLp3JxO0UqVd54PJbOMVTss-FqXLqL3I/edit
+Link to Google Drive of our engine : https://drive.google.com/drive/folders/13uqdCMKSp5BYv3FkD2sflDcJ6F_QaaVE
```

Comment:
```suggestion
```suggestion
[Production Pipeline document](https://docs.google.com/document/d/16_K5LYEYgaHoLp3JxO0UqVd54PJbOMVTss-FqXLqL3I/edit)
[Google Drive of the engine](https://drive.google.com/drive/folders/13uqdCMKSp5BYv3FkD2sflDcJ6F_QaaVE)
```